inntekt/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/inntekt/InntektLøser.kt

```diff
@@ -95,17 +99,27 @@ class InntektLøser(rapidsConnection: RapidsConnection, val inntektKlient: Innte
             val inntektResponse = hentInntekt(fnr, inntektPeriode, "helsearbeidsgiver-im-inntekt-$uuid")
             sikkerlogg.info("Fant inntektResponse: $inntektResponse")
             val inntekt = mapInntekt(inntektResponse, orgnr)
-            packet.setLøsning(BEHOV, InntektLøsning(inntekt))
+            packet.setLøsning(INNTEKT, InntektLøsning(inntekt))
             context.publish(packet.toJson())
             sikkerlogg.info("Fant inntekt $inntekt for $fnr og orgnr $orgnr")
         } catch (ex: Exception) {
             logger.error("Feil!", ex)
-            packet.setLøsning(BEHOV, InntektLøsning(error = Feilmelding("Klarte ikke hente inntekt")))
+            packet.setLøsning(INNTEKT, InntektLøsning(error = Feilmelding("Klarte ikke hente inntekt")))
             sikkerlogg.info("Det oppstod en feil ved henting av inntekt for $fnr", ex)
             context.publish(packet.toJson())
         }
     }
 
+    private fun hentOppdatertInntekt(dato: JsonNode): List<Periode> {
```

Comment:
Her synes jeg det hadde vært bedre å lest ut datoen et annet sted, for å slippe å sende rundt JsonNode.